platform/engineering/design-docs/2020-12-10-auto-deploy-cms-build.md

```diff
@@ -115,39 +101,21 @@ stage('Autodeploy content') {
 }
 ```
 
-#### Using the CMS export to build content
-
-In order for the content-only auto-deploy to use the CMS export data, the [build command](https://github.com/department-of-veterans-affairs/vets-website/blob/master/jenkins/common.groovy#L189) in the `build` function in `jenkins/common.groovy` will have to include the `--use-cms-export` flag.
-
 As it's a content-only deploy, it will build using the latest release instead of the latest commit of of `vets-website`.
 - This matches the behavior of the current content-only deploy.
 - The daily production deploy will continue deploying from the latest commit and retain the responsibility of tagging releases.
 
-If this work is done before the full build has transitioned to using the CMS export, the inclusion of the flag will be contingent on the `contentOnly` parameter to the function.
-
 #### Validating the build
 
-The implementation of the full CMS export build pipeline is expected to introduce a step that compares the diffs between GraphQL and CMS export builds.
-- That validation should be included in this pipeline as well while it's being rolled out.
-- The comparison step will be removed once the CMS export build is fully released.
-
 For now, the job will not retry or re-queue upon failure.
 - Such failures might be related to build diffs, broken links, or other validations.
 - The goal is for the job to recur at more frequent intervals, so we can look for a successful run in the next scheduled deploy.
 
-#### Deploying the build
-
-The pipeline will archive the CMS export build in S3 at a different key than the current GraphQL builds.
-
-For example, it might be `s3://vetsgov-website-builds-s3-upload/cms-export/${ref}/${envName}.tar.bz2`, where `ref` is the commit hash and `envName` is the environment (one of `vagovdev`, `vagovstaging`, or `vagovprod`).
-
-The subpath `cms-export` in the key will namespace the archived CMS export build separately from the current GraphQL build. This will enable testing and debugging of the new pipeline without intefering with the current deploy process.
-
 #### Scheduling the deploy
 
 The pipeline will [use a cron trigger](https://www.jenkins.io/doc/book/pipeline/syntax/#triggers) to run on the desired schedule.
 
-The initial schedule will be hourly between 8am and 8pm on weekdays. Ideally, the deploy would run every 5 minutes.
+The initial schedule will be hourly between 8am and 8pm on weekdays. Ideally, after incrementally increasing the deploy frequency, the deploy would run every 5 minutes.
```

Comment:
@theodur we're going to roll it out gradually instead:

**Schedule**:

- daily for 1-2 days
- 3 times a day for 1-2 days
- Hourly 8AM - 8PM Eastern

Reply:
I added this in there in a new commit